apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagsendringshendelseService.kt

```diff
@@ -98,7 +110,7 @@ class GrunnlagsendringshendelseService(
                     "Kunne ikke ferdigstille oppgaven for hendelsen på grunn av feil",
                     e,
                 )
-                throw e
+                throw KunneIkkeLukkeOppgaveForhendelse(e.message ?: "Kunne ikke ferdigstille oppgaven for hendelsen på grunn av feil")
```

Comment:
Kunne denne vært bedre? Dette feiler vel enten hvis det ikke er en åpen oppgave for hendelsen, eller vi prøver å ta den fra en annen saksbehandler

Reply:
nvm